lib/sync-DataSetModel.js

```diff
@@ -407,7 +456,7 @@ var self = {
       if (err) return cb(err);
 
 
-      var hashes = [];
+      /*var hashes = [];
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ACommentedCode)